### PR TITLE
[DDW-550] Hover not working properly on the first tile in the delegation center

### PR DESCRIPTION
**Pull request by yakovkaravelov**
This PR adjusted hover PopOver styles on the first tile in the delegation center screen
[Jira Ticket](https://jira.iohk.io/browse/DDW-550)

## Todos
- [x] Adjust hover PopOver styles on the first tile in the delegation center screen

## Testing Checklist
- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1613729133296700)

### Test Cases
```Gherkin
Given that I have a wallet fully synced
And I navigate to wallets tab of delegation centre
And the wallet is currently delegated and earning rewards in this epoch
When I hover over the first tile for the current epoch 
Then I can confirm the currently earning rewards tooltip pops up for the whole tile and not just stake pool ticker
```

### App
- [x] Go to `Staking` => `Delegation Center`, hover the first tile and make sure the PopOver styles are correct

### English
image

### Japanese
image

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Review by ManusMcCole — APPROVED** [review]
Looks good 👍

**Review by nikolaglumac — CHANGES_REQUESTED** [review]
@yakovkaravelov the tooltip's tip (triangle) is not correctly positioned when the number of characters in the stake pool ticker is different than 4...

Screenshot 2021-02-19 at 13 48 51
Screenshot 2021-02-19 at 13 48 53

Please implement better positioning for it as the char length and the width of chars in the ticker can vary...

**Comment by nikolaglumac** [comment]
@yakovkaravelov please let us know once this is ready to be tested again...

**Comment by yakovkaravelov** [comment]
@nikolaglumac @daniloprates @ManusMcCole @miorsufianiohk Please review/test, thanks.

**Comment by nikolaglumac** [comment]
@DominikGuzei please review this one 🙏

File: source/renderer/app/components/staking/delegation-center/WalletRow.js [hunk 1]
```diff
@@ -119,6 +120,48 @@ export default class WalletRow extends Component<Props, WalletRowState> {
...
+
+  constructor(props: Props) {
+    super(props);
+
+    this.stakePoolFirstTileRef = React.createRef();
+    this.stakePoolAdaSymbolRef = React.createRef();
+  }
+
+  componentDidUpdate() {
+    const {
```
Comment (nikolaglumac) [inline source/renderer/app/components/staking/delegation-center/WalletRow.js:134]: @yakovkaravelov can you please extract this logic into a separate method which then gets invoked in the `componentDidUpdate`?
That would be cleaner...

**Review by nikolaglumac — APPROVED** [review]
👌

**Review by ManusMcCole — APPROVED** [review]

**Pull request merged**